pkg/apis/monitoring/v1/podmonitor_types.go

```diff
@@ -207,6 +207,11 @@ type PodMetricsEndpoint struct {
 	// +optional
 	HonorTimestamps *bool `json:"honorTimestamps,omitempty"`
 
+	// TrackTimestampsStaleness whether Prometheus tracks staleness of
+	// the metrics that have an explicit timestamps present in scraped data.
```

Comment:
Is it worth adding `Has no effect if no timestamps are exposed or `honorTimestamps` is false.`?